pkg/runtime/grpc/grpc-runtime.go

```diff
@@ -172,6 +172,12 @@ func (r *Runtime) GlobalParamDescs() params.ParamDescs {
 				DefaultValue: fmt.Sprintf("%d", api.GadgetServicePort),
 				TypeHint:     params.TypeUint16,
 			},
+			{
+				Key:          utils.ParamGadgetNamespace,
```

Comment:
I actually don't know anymore why I did it this way.

Probably because `grpcruntime` can not be accessed from `kubectl-gadget/utils` module. The reason is that `grpcruntime` has a preexisting dependency on `kubectl-gadget/utils`.

I changed it and hardcoded the default namespace values there

Reply:
Now these 2 modules are decoupled